app/router.js

```diff
@@ -3,7 +3,19 @@ import config from './config/environment';
 
 const Router = Ember.Router.extend({
     location: config.locationType,
-    rootURL: config.rootURL
+    rootURL: config.rootURL,
+    willTransition(oldInfos, newInfos) {
+        const disableList = ['settings', 'moderation'];
```

Comment:
This should be an all-caps constant at the top level of the file, with a more descriptive name (maybe `SCROLL_RESET_DISABLED_ROUTES`? or something more concise)